apps/web/src/components/molecules/StudyDetails/StudyDetails.tsx

```diff
@@ -1,61 +1,113 @@
 import { Table } from '@nihr-ui/frontend'
 
+import type { getStudyByIdFromCPMS } from '@/lib/cpms/studies'
 import type { getStudyById } from '@/lib/studies'
 
 export interface StudyDetailsProps {
   study: NonNullable<Awaited<ReturnType<typeof getStudyById>>['data']>
+  studyInCPMS: NonNullable<Awaited<ReturnType<typeof getStudyByIdFromCPMS>>['study']>
 }
 
-export function StudyDetails({ study }: StudyDetailsProps) {
+function normalizeStudyData(study: StudyDetailsProps['study'], studyInCPMS: StudyDetailsProps['studyInCPMS']) {
+  const {
+    title: studyTitle,
+    protocolReferenceNumber: studyProtocol,
+    irasId: studyIrasId,
+    cpmsId: studyCpmsId,
+    organisationsByRole: studyOrgs,
+    managingSpeciality: studyManagingSpecialty,
+    chiefInvestigatorFirstName: studyCIFirstName,
+    chiefInvestigatorLastName: studyCILastName,
+    route: studyRoute,
+  } = study
+
+  const {
+    Title: cpmsTitle,
+    ProtocolReferenceNumber: cpmsProtocol,
+    IrasId: cpmsIrasId,
+    StudyId: cpmsStudyId,
+    organisationsByRole: cpmsOrgs,
+    ManagingSpecialty: cpmsManagingSpecialty,
+    ChiefInvestigatorFirstName: cpmsCIFirstName,
+    ChiefInvestigatorLastName: cpmsCILastName,
+    StudyRoute: cpmsRoute,
+  } = studyInCPMS
+
+  let chiefInvestigator = 'None available'
+
+  if (cpmsCIFirstName && cpmsCILastName) {
+    chiefInvestigator = `${cpmsCIFirstName} ${cpmsCILastName}`
+  } else if (studyCIFirstName && studyCILastName) {
+    chiefInvestigator = `${studyCIFirstName} ${studyCILastName}`.trim()
+  }
+  return {
+    title: cpmsTitle || studyTitle,
+    protocolReferenceNumber: cpmsProtocol || studyProtocol,
+    irasId: cpmsIrasId || studyIrasId,
+    cpmsId: cpmsStudyId || studyCpmsId,
+    organisationsByRole: cpmsOrgs || studyOrgs,
+    managingSpecialty: cpmsManagingSpecialty || studyManagingSpecialty,
+    chiefInvestigator,
+    studyRoute: cpmsRoute || studyRoute,
+  }
+}
+
+export function StudyDetails({ study, studyInCPMS }: StudyDetailsProps) {
+  const normalizedStudyData = normalizeStudyData(study, studyInCPMS)
+
   return (
     <Table className="govuk-!-margin-bottom-3">
       <Table.Caption className="govuk-visually-hidden">About this study</Table.Caption>
       <Table.Body>
         <Table.Row>
           <Table.CellHeader className="w-1/3">Study full title</Table.CellHeader>
-          <Table.Cell>{study.title}</Table.Cell>
+          <Table.Cell>{normalizedStudyData.title}</Table.Cell>
         </Table.Row>
-        {study.route === 'Commercial' && (
+
+        {normalizedStudyData.studyRoute === 'Commercial' && (
           <Table.Row>
             <Table.CellHeader className="w-1/3">Protocol reference number</Table.CellHeader>
-            <Table.Cell>{study.protocolReferenceNumber ?? 'None available'}</Table.Cell>
+            <Table.Cell>{normalizedStudyData.protocolReferenceNumber}</Table.Cell>
```

Comment:
Do we need the falsy check to show the default value here as I don't see this being set in the normalise function above?